### ISSUE TITLE
Invalid path after conversion at `git:init`

### ISSUE THREAD

**Issue by igormukhingmailcom** [phpro/grumphp#31]
Hi.

After running command:
```
php ./vendor/bin/grumphp git:init --config=vendor/linkorb/convensions-checker/config/app/grumphp.yml
```

I have next `.git/hooks/post-commit`:

```
#!/bin/sh

#
# Run the hook command.
# Note: this will be replaced by the real command during copy.
#
(cd "./" && exec 'vendor/bin/grumphp' 'git:pre-commit' '--config=../../../../../..') # <------------ MUST BE something like '--config=vendor/linkorb/convensions-checker/config/app/grumphp.yml'
...
```

What i'm doing wrong?
Thank you.

**Comment by veewee** [comment]
Hello @igormukhingmailcom,

Just tried following commands in a clean folder:
```
 git init
 composer require phpro/grumphp
 mkdir -p vendor/linkorb/convensions-checker/config/app/
 mv grumphp.yml vendor/linkorb/convensions-checker/config/app/grumphp.yml
 php ./vendor/bin/grumphp git:init --config=vendor/linkorb/convensions-checker/config/app/grumphp.yml
 cat .git/hooks/pre-commit
```

The results look like: 
```
(cd "./" && exec 'vendor/bin/grumphp' 'git:pre-commit' '--config=vendor/linkorb/convensions-checker/config/app/grumphp.yml')
```

Which is exactly what I would suspect it should be.
Can you give me some additional information about the problem?

**Comment by igormukhingmailcom** [comment]
Thank you very much for fast reply.

I've found problem. 
When grumphp can't find config at path from `--config` (this is another story why that file wasn't there) - it use default config without any warnings. So, it write path to default config, placed at project root directory. 

That would be great if grumphp will write some warning if file from `--config` doesn't exists.
Thanks a lot.

**Comment by veewee** [comment]
Fixed by #34
